pkg/convert/testdata/programs/implicit_required_providers/pcl/diagnostics.json

```diff
@@ -0,0 +1,4 @@
+[
+  "warning:implicit_required_providers/main.tf:5,1-48:Failed to get provider info:Failed to get provider info for \"tfe_organization\": could not find mapping information for provider tfe; try installing a pulumi plugin that supports this terraform provider",
```

Comment:
Why do we get mapping warnings? If we know we're using the parameterised version can't we lookup the mapping from that?

Reply:
It felt weird to add local mappings for a provider for which we load mappings on the fly. I added them now but I think the tests don't account well for parameterized providers